sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 4.0.0-beta.1 (Unreleased)
+## 4.0.0-beta.1 (2023-02-07)
 
 - initial preview of Settings API. 
```

Comment:
Please add this under the `Features Added` heading like the other changelog entries.

```suggestion
### Features Added

- Initial preview of Settings API. 
```

Reply:
Without that section, it might be even possible that release CI will fail.